docs/resources/runbook.md

```diff
@@ -62,6 +62,25 @@ resource "firehydrant_runbook" "example-runbook" {
     automatic        = false
     repeats          = true
     repeats_duration = "PT15M"
+    rule = jsonencode({
+      logic = {
+        eq = [
+          {
+            var = "incident_current_milestone",
+          },
+          {
+            var = "usr.1"
+          }
+        ]
+      },
+      user_data = {
+        "1" = {
+          type  = "Milestone",
+          value = "resolved",
+          label = "Resolved"
+        }
+      }
+    })
```

Comment:
very tiny nit: can you get rid of the extra commas in the doc and test examples?
```suggestion
    rule = jsonencode({
      logic = {
        eq = [
          {
            var = "incident_current_milestone"
          },
          {
            var = "usr.1"
          }
        ]
      }
      user_data = {
        "1" = {
          type  = "Milestone"
          value = "resolved"
          label = "Resolved"
        }
      }
    })
```